src/Logger.ts

```diff
@@ -68,6 +72,16 @@ class Logger {
         this.log(Logger.Level.SUCCESS, message, ...obj);
     }
 
+    /**
+     * Check if a log level should be logged
+     * @param level Log level
+     * @returns true if the log level should be logged
+     */
+    public shouldLog(level: Logger.Level): boolean {
+        return Logger.LevelHierarchy.indexOf(level)! >= Logger.LevelHierarchy.indexOf(this.logLevel!)!;
```

Comment:
```suggestion
        return Logger.LevelHierarchy.indexOf(level) >= Logger.LevelHierarchy.indexOf(this.logLevel);
```

not sure if this can be null/undefined